reverse_engineering/api.js

```diff
@@ -21,19 +21,22 @@ const { prepareError } = require('./databaseService/helpers/errorService');
 
 module.exports = {
 	async connect(connectionInfo, logger, callback, app) {
-		const client = getClient();
-		const sshService = app.require('@hackolade/ssh-service');
+		const client = clientManager.getClient();
+
 		if (!client) {
-			await setClient(connectionInfo, sshService, 0, logger);
-			return getClient();
+			await clientManager.setClient({
```

Comment:
`initClient()` would be semantically more correct than `setClient()` since you don't pass the client but rather the parameters that the manager needs to initialise a client.